From b9d1970ad0c2b9209bd67a9127667bdb6145b08c Mon Sep 17 00:00:00 2001
From: Andrew Cooper <andrew.cooper@novatechweb.com>
Date: Thu, 10 Jan 2019 16:31:45 -0600
Subject: [PATCH] Update configure for modern sed and python

---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 7a34e7a8a..e00343b18 100644
--- a/configure.ac
+++ b/configure.ac
@@ -125,7 +125,7 @@ if test -z "$SED"; then
 fi
 
 AC_MSG_CHECKING([sed version])
-SED_VERSION=`$SED --version 2>/dev/null | $SED -ne "s/.*version \([[0-9\.]]*\)/\1/p"`
+SED_VERSION=`$SED --version 2>/dev/null | $SED -ne "s/.*sed (GNU sed) \([[0-9\.]]*\)/\1/p"`
 case "$SED_VERSION" in
 3.*) AC_MSG_ERROR([we need at least GNU sed 4.x but found $SED_VERSION]) ;;
 4.*) ;;
@@ -350,7 +350,7 @@ else
 fi
 
 AC_MSG_CHECKING(whether python development files are present)
-PYTHON_CONFDIR=`$PYTHON -v -c "import distutils" 2>&1 | $EGREP distutils | $EGREP directory | $SED -ne "s/^[[^\/]]*\([[^[:space:]]]*\).*/\1/p"`/../config/
+PYTHON_CONFDIR=`$PYTHON -v -c "import distutils" 2>&1 | $EGREP distutils | $EGREP directory | $SED -ne "s/^[[^\/]]*\([[^[:space:]]]*\).*/\1/p"`/../config-x86_64-linux-gnu/
 if test -e "$PYTHON_CONFDIR/Makefile"; then
 	AC_MSG_RESULT(yes)
 else
-- 
2.17.1

